web.server.rules

```diff
@@ -0,0 +1,6 @@
+MainRule  "rx:^[a-zA-Z\d-]+\.[a-zA-Z]+$" "msg:HOST-Header Injection" "mz:$HEADERS_VAR:Host" "s:$ATTACK:6" id:42000465 ;
+MainRule  "rx:<!DOCTYPE(\s+)(%*\s*)([{}:.a-zA-Z0-9_-]*)(\s+)SYSTEM" "msg: possible XML/XXE-Exploitation atempt (Doctype)" "mz:BODY" "s:$ATTACK:8" id:42000455  ;
+MainRule  "str:meterpreter" "msg:Meterpreter-UA detected" "mz:$HEADERS_VAR:User-Agent" "s:$ATTACK:8" id:42000381  ;
+MainRule  "str:/.git/" "msg:SSH-Homedir-Access" "mz:URL" "s:$ATTACK:8" id:42000329  ;
```

Comment:
I don't quite understand the link between a .git directory and SSH-Homedir-Access. Am I missing something or could the message be more explicit here ?

Reply:
i think is a mistype.